tests_metricflow/snapshots/test_offset_metrics.py/str/DuckDB/test_offset_window_with_date_part_only__query_output.txt

```diff
@@ -0,0 +1,9 @@
+test_name: test_offset_window_with_date_part_only
```

Comment:
also doesn't really make sense... remove? wait, didn't this fail? did I already remove it?

Reply:
yes but why did this work and not grain to date? check the rules